### PR TITLE
.gitignore records for build artifacts #2

### PR DESCRIPTION
**Pull request by denis-tingajkin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

## Description
Related to https://github.com/networkservicemesh/networkservicemesh/pull/1664

## Motivation and Context
Adds missed `cmd` artifact to `.gitignore`

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,4 @@ examples/
 build/dist
 scripts/aws/aws
 controllers/sriov-controller/sriov-controller
+cmd
```
Comment (haiodo) [inline .gitignore]: Why we need this?

Comment (denis-tingajkin) [reply .gitignore]: ~3s
`<testcase classname="github.com/networkservicemesh/networkservicemesh/test/cloudtest/pkg/utils" name="TestProcessOutputShouldNotLostOutput" time="2.630000"/>` from JunitResult

Comment (denis-tingajkin) [reply .gitignore]: I have the untracked `cmd` binary in the repo after I did sh scripts/for-each-module.sh "go build ./..."

Comment (denis-tingajkin) [reply .gitignore]: It this my local problem?

Comment (denis-tingajkin) [reply .gitignore]: will check on another machine. Marked PR as WIP

**Pull request merged**